packages/SwingSet/test/test-message-patterns.js

```diff
@@ -3,8 +3,11 @@
 /* eslint dot-notation: "off" */
 /* eslint object-shorthand: "off" */
 
-import '@agoric/install-ses';
+// `test.repeat` does not yet seem compatible with ses-ava
+// import { test } from '../tools/prepare-test-env-ava';
```

Comment:
This may be true but as best I can determine nothing here (or anywhere in `Swingset/test` for that matter) uses `test.repeat` so what is the issue with this particular test file?

Reply:
Doh! I remembered the wrong name. `test.serial` is the problem. See https://github.com/endojs/endo/issues/647